graph/model/user.go

```diff
@@ -2,6 +2,10 @@ package model
 
 import "time"
 
+const (
+	ConciergeUser = "88c8c5ae-41e9-49ef-b5ea-db2e08762693"
```

Comment:
Any reason we wouldn't want this to differ across environment? I can't really imagine a true reason to keep these different other than safety. Like someone accidentally thinking they're working in dev but working in prod.

Reply:
the safety angle makes sense. do we want to add this to the config?